src/__tests__/__snapshots__/csvExport.js.snap

```diff
@@ -1,398 +1,398 @@
-2018-02-21T18:24:28.300Z,BCH,IN,0.02116023,0.0002116,9A2ECBBEE6FF5623F8683ED3FD6EC9487AEE2A3244D3E7F6FD7EE928195DBCDA,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-21T18:15:28.930Z,BCH,OUT,0.00139705,0.00001397,881140BB77AC2EC169B0FFCD2E6F0FC3DE1D325F607DF242DB8E8B0DA21B00EE,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-21T15:45:06.301Z,BCH,IN,0.06073771,0.00060738,F3A7B6144EBA4AD35A4596D2FF07E90026115CA6CF991A445B07720DDDD2210A,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-21T12:43:18.521Z,BCH,IN,0.00250447,0.00002504,F17932066CD4A9F37E499DBB86615323723BEFC9E2D6802AEC3826AD6E02EC3D,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-20T18:40:38.537Z,BCH,IN,0.4842576,0.00484258,67B4E34A05C543B009967EA5AE67D89CC6E5A5FB6B091B633006F7E657FF1052,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-20T13:52:45.869Z,BCH,IN,0.03281369,0.00032814,1379D7E5CF02028A132A4F0D5D781A725AEA9B5DDB74933AC530AFC03EC9A0F2,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-20T13:05:53.970Z,BCH,IN,0.2921288,0.00292129,DD5E88A322D61260A5FF67B118B5332295135AB4E29EC9EA60D3F76BE3E151B1,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-20T12:46:00.726Z,BCH,OUT,0.09427549,0.00094275,74DD30C30D99E54E0609B52FD072316F7586ABC000EA1580E1C3FB34E74B1583,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-20T02:02:07.198Z,BCH,IN,0.13346111,0.00133461,F1009880774DB5F34067130ABFAFF562F0F8E083AC59074E417BE7F20C458B48,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-19T23:56:05.291Z,BCH,IN,0.13699633,0.00136996,33AC8491444FBF6E9EFAC9FB2D2A73437E2D3E66D7AD7376AE59EA941A13A1B9,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-19T22:18:15.952Z,BCH,IN,0.0005963,0.00000596,A9C100D4CEBE736B178092A87F7CB4B2DE43217936A45D50E0BA7ADF7D3245D9,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81
-2018-02-19T16:47:58.776Z,BCH,IN,0.00305818,0.00003058,1DEBA260535EBACF84DBCCB6E2DCF1012400E2CE9FBF7477442B07D0E4794F26,SaOdFlBkKWyIpOTKRYNuUKt4,215F2A26F71261B2717BB6191D592D7CE8FF1F2F9BA8BF89448B31062741CB81"
+"Operation Date,Currency Ticker,Operation Type,Operation Amount,Operation Fees,Operation Hash,Account Name,Account xpub,Countervalue Ticker,Countervalue at Operation Date,Countervalue at CSV Export
+2018-03-14T17:32:01.367Z,ZEC,IN,0.51754633,0.00517546,5F1D58875436D84ACE9E9648AE611384EEA7E23DF310006E5E69746913FE3128,gqLASLJoyCjGiQ0qUZVPJCP5KV8bhdtZw,F3320BB853F7E4E8A9BCE3CF4CF75EAA119A06DAD660340907D03EA21B6DB162,USD,,
```

Comment:
mmh snapshot shows no countervalues at all.

Reply:
yeah there was an issue in the test where the mock accounts were not taking in param the currencies and creating random currency accounts, just updated it :bow: